### PR TITLE
Add note when using count greater than one on relationships

### PR DESCRIPTION
**Pull request by damsfx**

### CODE REVIEW

File: database/relations.md [hunk 1]
```diff
@@ -791,6 +791,8 @@ You may also specify an operator and count to further customize the query:
 $posts = Post::has('comments', '>=', 3)->get();
 ```
 
+> **NOTE**: With a MySQL database, you may experience an error when performing a count greater than one. In this case, you can set the `strict` option to **false** in your database configuration `config/database/connections/mysql/strict`.
```
Comment (LukeTowers) [inline database/relations.md]: What specifically is the error?

Comment (damsfx) [reply database/relations.md]: According to the Winter docs : 
```php
// Retrieve all posts that have three or more comments...
$posts = Post::has('comments', '>=', 3)->get();
```

So my test code : 
```php
$posts = Post::has('featured_images', '>=', 1)->get();  // Ok
$posts = Post::has('featured_images', '>=', 2)->get();  // Not Ok, issue
```
The issue : 
`Syntax error or access violation: 1140 Mixing of GROUP columns (MIN(),MAX(),COUNT(),...) with no GROUP columns is illegal if there is no GROUP BY clause (SQL: select * from winter_blog_posts where (select count(*) from system_files where system_files.attachment_id = winter_blog_posts.id and system_files.attachment_type = Winter\Blog\Models\Post and field = featured_images order by sort_order asc) >= 2)`

You suggested setting the strict mode to false in the database configuration to solve this.
see: 
https://discord.com/channels/816852513684193281/821159568763125800/1221846556462809130

Comment (LukeTowers) [inline database/relations.md]:
```suggestion
> **NOTE**: MySQL strict mode can sometimes complain when using a `GROUP` column without a `GROUP BY` clause (in the above case, `COUNT()`). You can set the `strict` option to `false` in your database's connection configuration (i.e. `database.connections.mysql.strict`) to ignore this warning message.
```

Comment (damsfx) [reply database/relations.md]: So much better!

**Pull request merged**